components/pixels/lib.rs

```diff
@@ -257,6 +291,48 @@ fn is_webp(buffer: &[u8]) -> bool {
     buffer[8..].len() >= len && &buffer[8..12] == b"WEBP"
 }
 
+fn gif_decode(buffer: &[u8], cors_status: CorsStatus) -> Option<Image> {
```

Comment:
`decode_gif` so that the verb goes first here:

```suggestion
fn decode_gif(buffer: &[u8], cors_status: CorsStatus) -> Option<Image> {
```